tests/upgrading/README.md

```diff
@@ -31,8 +31,8 @@ Similar to the `provisioning` tests, the node scaling tests have static test cas
 
 ```yaml
 terratest:
-  kubernetesVersion: "v1.26.10+k3s2"
-  upgradedKubernetesVersion: "v1.27.7+k3s2"
+  kubernetesVersion: "v1.26.11+k3s2"
```

Comment:
is it k3s2 or k3s1 for this version?

Reply:
When I was testing, I grabbed a version available found in Rancher and this was the version.